.github/workflows/main.yml

```diff
@@ -11,13 +11,13 @@ jobs:
       # Install NPM dependencies, cache them correctlyand run all Cypress tests
       - name: set master url
         if: github.ref == 'refs/heads/master'
-        run: CYPRESS_BASE_URL=http://rounds.studio
+        run: rm -rf node_modules && yarn cache clean && yarn && CYPRESS_BASE_URL=http://rounds.studio
```

Comment:
@muganwas Not sure if this helped, open to whatever, but had a hell of a time when I updated a package and ran the steps

This basically does a manual clean build - but I'd think github would do that anyways 🤔

Reply:
github does a clean install every time I don't think it helped, I'll also setup a different workflow, we'll see how it behaves after you get that merged.

Reply:
ok, I'll remove these changes and see if it passes. Thanks!

Reply:
Ok, but even though it fails, it's most something else causing the failure. I'd leave it the way it is because it doesn't matter anyway.